### DISCUSSION TITLE
test pushover to make sure its working?

### DISCUSSION THREAD

**Discussion by ideal2545** [jef/streetmerchant#1286] [Help] [answered]
Is there anyway to do a quick test with pushover, pushbullet or email just to confirm your settings are correct?

**Comment by R3PTIL3** [comment, marked as answer]
Yes.
npm run test:notification
👍

**Reply by ideal2545** [reply]
thanks, apparently i skipped that part!

**Reply by R3PTIL3** [reply]
My mistake 
test:notification 
is correct.
